src/labels/CustomLabels.labels

```diff
@@ -8662,6 +8662,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Failure message if an invalid base64 string was provided</shortDescription>
         <value>The base64 format is invalid. The number of characters must be a multiple of 4 and can only contain these characters: [a-z A-Z 0-9 + / =]</value>
     </labels>
+    <labels>
+        <fullName>geProceedAnyway</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Proceed Anyway</shortDescription>
```

Comment:
```suggestion
        <shortDescription>Process Anyway</shortDescription>
```

Reply:
@melissabarber - the mockup from Lauren says "Proceed" and not "Process" - just want to make sure we want to go with "process"?

https://salesforce.invisionapp.com/share/CBZUHFUYFH7#/screens/448085214_Error_State_-_Token_Expired_Revisits

Reply:
Yep, decided to go with '"Process" after discussing with the docs UI text committee. @laurenchen-sf please update the mockup.

Reply:
Thanks for checking @ptewson-sfdo!